apps/coscrad-frontend/src/store/slices/resources/spatial-features/types/spatial-feature-index-state.ts

```diff
@@ -0,0 +1,3 @@
+import { IIndexQueryResult, ISpatialFeatureViewModel } from '@coscrad/api-interfaces';
+
+export type SpatialFeatureIndexState = IIndexQueryResult<ISpatialFeatureViewModel>;
```

Comment:
We might want to go back and refactor the others to be like this.

Reply:
There's not a lot of value in this. Let's just 'strangle out' the old pattern as we come across it.